build/rollup-config.js

```diff
@@ -7,42 +7,48 @@ import pkg from '../package.json';
 import {createBanner} from './banner';
 
 const release = process.env.NODE_ENV === 'release';
+const watch = process.argv.indexOf('-w') > -1 || process.argv.indexOf('--watch') > -1;
 // Skip the git branch+rev in the banner when doing a release build
 const version = release ? pkg.version : `${pkg.version}+${gitRev.branch()}.${gitRev.short()}`;
 const banner = createBanner(version);
 
 const outro = `var oldL = window.L;
-exports.noConflict = function() {
+leaflet.noConflict = function() {
 	window.L = oldL;
 	return this;
 }
-
 // Always export us to window global (see #2364)
-window.L = exports;`;
+window.L = leaflet;`;
 
 /** @type {import('rollup').RollupOptions} */
-export default {
+const config = {
 	input: 'src/Leaflet.js',
 	output: [
 		{
 			file: pkg.main,
 			format: 'umd',
-			name: 'L',
+			name: 'leaflet',
 			banner: banner,
 			outro: outro,
 			sourcemap: true,
 			freeze: false,
 			esModule: false
-		},
+		}
+	],
+	plugins: [
+		release ? json() : rollupGitVersion()
+	]
+};
+
+if (!watch) {
```

Comment:
Why would we not want to build this during watch mode?

Reply:
Why not? Like already described I was developing in Leaflet and checked the function `noConflict` and it was not existing because of the watch mode. It would be good to keep the watch and release builds similar, else it can be confusing or something can be missed.

Reply:
Why should it not be in the watch mode? Is there a good reason? The build time can't be the reason, because there is not a real difference

Reply:
Sorry misunderstood the context behinde your question. I don't know why we don't want to build this. This was already configurated like that. Maybe a build-time reason